amundsen-kube-helm/README.md

```diff
@@ -29,6 +29,7 @@ This is setup templates for deploying [amundsen](https://github.com/amundsen-io/
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| `clusterDomain`          | Cluster domain                                                                                            | `cluster.local` |
```

Comment:
Can you adjust the docs taking into consideration below remarks? 

1. we shouldn't put key names in `` (see below) as it stands out from rest of them
2. should type be `Cluster domain` or `string`?
3. perhaps `Cluster domain` was supposed to be a description, not type?

![image](https://user-images.githubusercontent.com/5680655/129016905-29c48288-070c-44fb-adb5-54a01ae11bfa.png)

Reply:
Address above comments.